src/policyserv_api.ts

```diff
@@ -0,0 +1,49 @@
+export class PolicyservApi {
+    public constructor(private readonly baseUrl: string, private readonly apiKey: string) {
+    }
+
+    public async createCommunity(communityName: string): Promise<string> {
+        const community = await this.doRequest<CommunityResponse>("POST", "/api/v1/communities/new", {
+            name: communityName,
+        });
+        return community.community_id;
+    }
+
+    private async doRequest<T>(method: string, path: string, body?: any): Promise<T> {
+        const req = await fetch(this.baseUrl + path, {
+            method: method,
+            headers: {
+                "Authorization": `Bearer ${this.apiKey}`
+            },
+            body: body ? JSON.stringify(body) : undefined,
+        });
+        if (req.status !== 200) {
+            const body = await req.text();
+            throw new Error(`Request failed with status ${req.status}: ${body}`);
```

Comment:
might be nice to add the url that failed in the error message?

Reply:
oh, yes. Good call - will add